### PR TITLE
Remove double `subject` in api/v1/accounts/relationships spec

### PR DESCRIPTION
**Pull request by mjankowski**
Presumably side effect of a previous rebase.

### CODE REVIEW

File: spec/requests/api/v1/accounts/relationships_spec.rb [hunk 1]
```diff
@@ -104,7 +104,6 @@ def expect_bob_item_three
 
     it 'returns JSON with correct data on cached requests too' do
       subject
-      subject
```
Comment (ClearlyClaire) [inline spec/requests/api/v1/accounts/relationships_spec.rb:107]: This is actually intentional, but probably warrants a comment! The test is about the cache behavior, so about two renders of the same page.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

File: spec/requests/api/v1/accounts/relationships_spec.rb (hunk 1 above)
Comment (mjankowski) [reply spec/requests/api/v1/accounts/relationships_spec.rb:107]: Actually, I'm not sure what the caching check is. It doesn't look like api/v1/accounts/relationships controller turns on caching, and it's not listed in the cache_spec array of endpoints to check.

I see how the previous controller spec just had one request made - https://github.com/mastodon/mastodon/commit/c451bbe249475b937906f8f4d85c1f1eb9804343#diff-e3b53f38ced7fe961c10e70f948775483ad9554419b1c3415b506c238e4c061cL80

And then when moved to a request spec (nice!) it's a double - https://github.com/mastodon/mastodon/commit/c451bbe249475b937906f8f4d85c1f1eb9804343#diff-50ab22c0a93a1444d0996cf37d6cc79aa37ec8fc4dd0177d4d5d3ff8ed13b889R106-R107

Is there anything here which would be present in the first response but not the second, or vice versa (headers?) ... or am I missing something obvious about what's being cached/checked?

Comment (ClearlyClaire) [reply spec/requests/api/v1/accounts/relationships_spec.rb:107]: > I see how the previous controller spec just had one request made - [c451bbe#diff-e3b53f38ced7fe961c10e70f948775483ad9554419b1c3415b506c238e4c061cL80](https://github.com/mastodon/mastodon/commit/c451bbe249475b937906f8f4d85c1f1eb9804343#diff-e3b53f38ced7fe961c10e70f948775483ad9554419b1c3415b506c238e4c061cL80)

There were actually two, the first one was in a `before` block: https://github.com/mastodon/mastodon/commit/c451bbe249475b937906f8f4d85c1f1eb9804343#diff-e3b53f38ced7fe961c10e70f948775483ad9554419b1c3415b506c238e4c061cL41

But I don't remember exactly what it was testing.

Comment (mjankowski) [reply spec/requests/api/v1/accounts/relationships_spec.rb:107]: Looks like the introduction of the check was here - https://github.com/mastodon/mastodon/pull/6527/files

I see now what you mean about the previous setup with the initial call in the subject ... I suspect what it was originally checking was the regression referenced in that PR where the introduction of lower level caching was erroneously changing the response to those requests, even when the id values in the params changed (subject had two, narrower test had just one). I'll see if I can isolate the expected differences there.

Comment (mjankowski) [reply spec/requests/api/v1/accounts/relationships_spec.rb:107]: Pushed a change which restores the double request, and adds some comments/specs around why it's there.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**